### PR TITLE
[iosxr]: update tags for integration testing

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/571
##### SUMMARY

Update network_cli tag for integration testcases

### CODE REVIEW

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

File: README.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
-
-
```
Comment (rohitthakur2590) [inline README.md:2]: do we really want these to remove?

**Comment by pabelanger** [comment]
We actually need to add tags for connection local, which means we need to move the network_cli tag down into iosxr_smoke/tasks/cli.yaml for example

File: tests/integration/targets/iosxr_user/tasks/main.yaml [hunk 2]
```diff
@@ -1,8 +1,3 @@
 ---
 - include: cli.yaml
-  tags:
```
Comment (pabelanger) [inline tests/integration/targets/iosxr_user/tasks/main.yaml:3]: we look to be missing tasks/cli.yaml update here.

File: tests/integration/targets/iosxr_banner/tasks/main.yaml [hunk 3]
```diff
@@ -1,8 +1,3 @@
 ---
 - include: cli.yaml
-  tags:
-    - cli
-
 - include: netconf.yaml
-  tags:
```
Comment (pabelanger) [inline tests/integration/targets/iosxr_banner/tasks/main.yaml:7]: we didn't update tasks/netconf.yaml here

File: tests/integration/targets/iosxr_config/tasks/cli_config.yaml [hunk 4]
```diff
@@ -14,3 +14,5 @@
   with_items: '{{ test_items }}'
   loop_control:
     loop_var: test_case_to_run
+  tags:
+    - cli_config
```
Comment (pabelanger) [inline tests/integration/targets/iosxr_config/tasks/cli_config.yaml:18]: network_cli

File: tests/integration/targets/iosxr_banner/tasks/main.yaml (hunk 3 above)
Comment (pabelanger) [reply tests/integration/targets/iosxr_banner/tasks/main.yaml:7]: ignore this, I am not correct

File: tests/integration/targets/iosxr_banner/tasks/main.yaml [hunk 5]
```diff
@@ -1,8 +1,3 @@
 ---
 - include: cli.yaml
-  tags:
-    - cli
```
Comment (pabelanger) [inline tests/integration/targets/iosxr_banner/tasks/main.yaml:4]: this should be network_cli

File: tests/integration/targets/iosxr_user/tasks/main.yaml (hunk 2 above)
Comment (pabelanger) [reply tests/integration/targets/iosxr_user/tasks/main.yaml:3]: looks like we can just switch this to network_cli

File: tests/integration/targets/iosxr_banner/tasks/main.yaml (hunk 5 above)
Comment (GomathiselviS) [reply tests/integration/targets/iosxr_banner/tasks/main.yaml:4]: This is already in place. 'cli' tag is removed.

**Comment by GomathiselviS** [comment]
recheck

**Pull request merged**